tests/unit/test_databroker_business_events.py

```diff
@@ -214,7 +214,7 @@ def test_business_events_metrics_constants_config_free(self):
         assert be_config["BusinessEvents.GenerateMetrics"] == "false"
         assert be_config["BusinessEvents.EnableHeartbeat"] == "false"
 
-    def test_business_events_metrics_constants_config_free(self):
+    def test_business_events_metrics_constants_config(self):
```

Comment:
LGTM. just curious, why the linter doesn't like `_free`. maybe we rename it to `_sandbox` ?

Reply:
It's because the previous method is named the same, if there's a better name for it I can modify it too